www/src/components/shell/Shell.js

```diff
@@ -52,8 +55,16 @@ function Shell({ shell }) {
     }
 
     term.write(`Booting into your ${shell.provider} shell...\r\n\r\nIt can take a few minutes to load. Try refreshing the page if it gets stuck for too long.\r\n`)
-    chan.onError(err => console.error(`Unknown error during booting into your shell: ${err}`))
-    chan.on('stdo', ({ message }) => term.write(decodeBase64(message)))
+    chan.onError(err => console.error(`Unknown error during booting into your shell: ${JSON.stringify(err)}`))
+    chan.on('stdo', ({ message }) => {
+      const decoded = decodeBase64(message)
+
+      term.write(decodeBase64(message))
```

Comment:
is there a reason we don't pass the decoded const here instead of doube b64 decoding?

Reply:
Definitely no reason. I forgot to update this line. Good catch :slightly_smiling_face: